Sources/Keychain.swift

```diff
@@ -38,16 +38,20 @@ public class OktaKeychain: NSObject {
     internal class func set(key: String, _ object: String) {
         let objectData = object.data(using: .utf8)
 
-        let q = [
+        var q: [String: Any] = [
             kSecClass as String: kSecClassGenericPassword as String,
             kSecValueData as String: objectData!,
             kSecAttrAccount as String: key
-        ] as CFDictionary
+        ]
+
+        if #available(macOS 10.15, iOS 13.0, *) {
```

Comment:
Can you change this macros for MacOS only. I would like to avoid potential regressions for existing iOS users

Reply:
Done!